cmd/juju/model/configcommand_test.go

```diff
@@ -106,7 +106,15 @@ func (s *ConfigCommandSuite) TestSingleValueOutputFile(c *gc.C) {
 
 func (s *ConfigCommandSuite) TestGetUnknownValue(c *gc.C) {
 	context, err := s.run(c, "unknown")
-	c.Assert(err, gc.ErrorMatches, `key "unknown" not found in {<nil> ""} model.`)
+	c.Assert(err, gc.ErrorMatches, `value: "unknown" seems to be neither a file nor a key of the currently targeted model: {<nil> \"\"}`)
+
+	output := cmdtesting.Stdout(context)
+	c.Assert(output, gc.Equals, "")
+}
+
+func (s *ConfigCommandSuite) TestGetProperErrorMessageOnPaths(c *gc.C) {
```

Comment:
if we go the route of the isFileLike should that have a series of unit tests for things we expect to be file-like vs not?

Reply:
hmm, yes makes sense! Iam taking a further look into this

Reply:
ty

Reply:
Added 2 additional tests. Others are already testing partly the functionality